dpe/src/commands/certify_key.rs

```diff
@@ -625,4 +626,66 @@ mod tests {
             assert!(extension.critical);
         }
     }
+
+    #[test]
+    fn test_certify_key_order() {
+        CfiCounter::reset_for_test();
+        let mut env = DpeEnv::<TestTypes> {
+            crypto: OpensslCrypto::new(),
+            platform: DefaultPlatform,
+        };
+        let mut dpe = DpeInstance::new(&mut env, Support::X509).unwrap();
+
+        let init_resp = match InitCtxCmd::new_use_default()
```

Comment:
Since we are auto-initing with a measurement of 0s, can we just add AUTO_INIT to the DPE's support? Seems like there is no need to provide an explicit measurement

Reply:
Do you mean doing auto init + 1 DeriveContext instead of 2 DeriveContexts? Or something else?

Reply:
Just meant that it seems like we don't need to call new_auto_init. Instead, we can just support AUTO_INIT. 

We could also just do 1 deriveContext with a tci_type of something other than 0.

Reply:
Got it, done.